api/ocm/internal/modopts.go

```diff
@@ -108,6 +108,10 @@ type TargetOptionImpl interface {
 
 type TargetElementOptions struct {
 	TargetElement TargetElement
+
+	// RawIdentity disables the implicit defaulting of the extraIdentity.
+	// A transfer operation must set this flag to preserve the normalizations.
+	RawIdentity *bool
```

Comment:
This field should be named after what it does, e.g. DisableImplicitExtraIdentityDefaulting

Reply:
The treason for `RawIdentity` was the the method `GetRawIdentity`, which returned the identity as given and not the effective identity using the implicit version adding wherever required.